gpkit/fmincon/simpleflight.py

```diff
@@ -0,0 +1,62 @@
+"Minimizes airplane drag for a simple drag and structure model."
```

Comment:
yeah, it's a bit icky, but you could probably do a relative import to the examples dir.